server/controllers/dashboardController.ts

```diff
@@ -1,7 +1,7 @@
 import type { Request, RequestHandler, Response } from 'express'
 import paths from '../paths/manage'
 
-export default class ApplicationController {
+export default class DashboardController {
```

Comment:
This is going to make finding the Applications controller so much easier!

Reply:
I considered keeping the name the same, but I'm glad I didn't now! 😆